aspnetcore/security/authorization/iauthorizationpolicyprovider.md

```diff
@@ -155,11 +156,11 @@ public Task<AuthorizationPolicy> GetDefaultPolicyAsync() =>
 
 As with all aspects of a custom `IAuthorizationPolicyProvider`, you can customize this, as needed. In some cases, it may be desirable to retrieve the default policy from a fallback `IAuthorizationPolicyProvider`.
 
-## Required policy
+## Fallback policy
 
-A custom `IAuthorizationPolicyProvider` needs to implement `GetRequiredPolicyAsync` to, optionally, provide a policy that is always required. If `GetRequiredPolicyAsync` returns a non-null policy, that policy will be combined with any other (named or default) policy that is requested.
+A custom `IAuthorizationPolicyProvider` must optionally implement `GetFallbackPolicyAsync` to provide a policy that's used by the Authorization Middleware when no policies are specified. If `GetFallbackPolicyAsync` returns a non-null policy, that policy is used by the Authorization Middleware when no policies are specified for the request.
 
-If no required policy is needed, the provider can just return null or defer to the fallback provider:
+If no fallback policy is required, the provider can return `null` or defer to the fallback provider:
```

Comment:
How is deferral done?

Reply:
In this particular sentence, its just talking about the custom provider implementation of the method, basically you can return null, or call into the default provider (which we are calling the fallback provider).  confusing :)